egs/aspire/s5/local/multi_condition/aspire_prep_rir_noise_list.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python
```

Comment:
We will most probably not use this script for long, as we soon plan to create an RIR database and upload it to openslr.org. This script is a stop gap arrangement to use the new `steps/data/reverberate_data_dir.py` until the new upload is prepared. So I am not going ensure that this script is very generic and I will allow for a lot of assumptions to made.